### DISCUSSION TITLE
ReOpen invalidate/delete sessions

### DISCUSSION THREAD

**Discussion by sergitejada** [nextauthjs/next-auth#9559] [Help]
I am using CredentialsProvider and Rails to store my users.
If I delete a user from my database (ie. I want to block their access to my app), next-auth doesn't force their session to get logged out. When that user refreshes the page, it lets them get all the way through because it continues to use their JWT cookie token.

**Comment by cexra** [comment]
After I searched for the same issue, I found a discussion about it. maybe this can help you https://github.com/nextauthjs/next-auth/discussions/4687

**Reply by sergitejada** [reply]
it's a good thread but there is no clear way to solve it.
There are a couple of other similar threads but none of them provide a solution.
